### ISSUE TITLE
FamilyType.ByGeometry shown blank family in Revit 2022

### ISSUE THREAD

**Issue by hardhiraf** [DynamoDS/DynamoRevit#2717]
## Dynamo version

2.10.1

## Revit version

Revit 2022

## Operating system

Windows 10

## What did you do? 

I wanna create family by geometry using FamilyType.ByGeometry node like I did a lot in Revit 2021.

![image](https://user-images.githubusercontent.com/59072757/122344486-0c53ba80-cf71-11eb-8d81-0c5b4f1db35e.png)

## What did you expect to see?

This is what it looks like in Revit 2021

![image](https://user-images.githubusercontent.com/59072757/122345533-26da6380-cf72-11eb-8307-4e16f358d853.png)

## What did you see instead?

This is what it looks like in Revit 2022

![image](https://user-images.githubusercontent.com/59072757/122344947-82582180-cf71-11eb-9569-7ec4e99d6d64.png)

**Comment by ZiyunShang** [comment]
Hi @hardhiraf , I just use a cuboid to create a family type.
![Cuboid-FamilyType](https://user-images.githubusercontent.com/33445445/124555163-57266b00-de69-11eb-856c-8b7aea9d73a9.PNG)
It can get a cube icon here.
I think this icon may be lost in Revit 2022.

**Comment by hardhiraf** [comment]
Thank you for your reply. But in my case, not just the icon, the family type was also not created.

If I create Family Instance from the Family Type in Dynamo
![DYN](https://user-images.githubusercontent.com/59072757/124598136-80a5bd80-de8e-11eb-9e43-262a71eaac35.PNG)

This is what it looks like in Revit 2021
![RVT 2021](https://user-images.githubusercontent.com/59072757/124598213-94512400-de8e-11eb-87f2-13cf1a0fabcf.PNG)
as you see, there is icon and family instance was created

and this is in Revit 2022
![RVT 2022](https://user-images.githubusercontent.com/59072757/124598420-c6628600-de8e-11eb-870a-6295fb2c9960.PNG)
there is no icon and family instance was not created

**Comment by ZiyunShang** [comment]
Hi @hardhiraf , sorry for late response. 
![Cuboid-FamilyInstance](https://user-images.githubusercontent.com/33445445/125224920-fe3f5280-e300-11eb-9e95-93150e545efd.PNG)

I use a simple solid can't reproduce your issue. If it is convenient, could you please share your file so that I can reproduce your problem.

**Comment by hardhiraf** [comment]
Thank you @ZiyunShang, This is my file
[FG SOLID.zip](https://github.com/DynamoDS/DynamoRevit/files/6798499/FG.SOLID.zip)

**Comment by ZiyunShang** [comment]
Hi @hardhiraf , the main reason for this issue is the change of Revit API - "Document.Import". Your SAT file was imported into Revit as Solid Geometry before 2022, but in 2022 it is imported not as Solid Geometry, but as Mesh.

We are seeking the help of the owner of this API to confirm whether this change is reasonable.

**Comment by hardhiraf** [comment]
Hi @ZiyunShang, thanks for the help. Maybe I'll close this issue if there is an answer from the owner of that API. I'll be waiting. Thanks again!

**Comment by hardhiraf** [comment]
Hi @ZiyunShang any update on this issue?

**Comment by ZiyunShang** [comment]
> Hi @ZiyunShang any update on this issue?

No, I have created problem report to the Revit API owner, but it hasn't been solved.

**Comment by hardhiraf** [comment]
Okay, thank you for your response.

**Comment by kub1x** [comment]
Hey, I tried to describe the problem in following thread and am hoping for finding a workaround to sort this problem out. I'm primarily a programmer and am not that much experienced in Revit. I'd appreciate support or further information on that forum to hopefully drive attention to it. 

https://forums.autodesk.com/t5/revit-api-forum/document-import-of-sat-file-behavior-changes-in-2022-1-imports/td-p/10833349

I didn't find link to the issue on the Revit API owner mentioned by @ZiyunShang so couldn't link or 👍 it.

**Comment by ZiyunShang** [comment]
Hi @kub1x , I’m not sure if you can open this link, if you need it, I’ll put it here - https://jira.autodesk.com/browse/REVIT-179667

**Comment by johnpierson** [comment]
Closing this out as part of a backlog cleanup. The thread went quiet years ago and the versions involved are well behind current.

If this still reproduces on a current Dynamo and Revit, please open a fresh issue with the graph and a sample model attached.
